repo.go

```diff
@@ -125,24 +133,27 @@ func (r *Repo) parseReference(ref string) (*plumbing.Hash, error) {
 // Note: While Fetch itself is thread-safe in that it ensures a previous Fetch() is completed before starting a new one,
 // the Repo is not. If Fetch is called from two go routines, subsequent reads may be non-deterministic.
 func (r *Repo) Fetch() error {
-	// Perform a fetch on the repository
-	err := r.fetch()
-	// Ignore "already-up-to-date" error
-	if err != nil && err != git.NoErrAlreadyUpToDate {
-		return fmt.Errorf("unable to fetch repository: %v", err)
-	}
-	return nil
+	return r.FetchContext(context.Background())
 }
 
-// fetch performs a fetch on the internal repository while under a lock
-func (r *Repo) fetch() error {
+// FetchContext performs a Git fetch of the repository.
```

Comment:
Very minor nit, but should we keep the `xxxxContext` and `xxxx` methods in the same order through the file?

Reply:
Ah yes, consistency is preferable :+1: